serverless-functions/src/functions/common/twilio-wrappers/taskrouter.private.js

```diff
@@ -499,3 +499,44 @@ exports.fetchTask = async function fetchTask(parameters) {
     return retryHandler(error, parameters, arguments.callee);
   }
 };
+
+/**
+ * @param {object} parameters the parameters for the function
+ * @param {number} parameters.attempts the number of retry attempts performed
+ * @param {object} parameters.context the context from calling lambda function
+ * @param {object} parameters.workflowSid (optional) the workflow SID to filter by
+ * @param {object} parameters.assignmentStatus (optional) the assignment status to filter by
+ * @param {object} parameters.ordering (optional) the desired ordering (e.g. DateCreated:desc)
+ * @returns {object} An object containing an array of tasks for the account
+ * @description the following method is used to robustly retrieve
+ *  tasks for the account
+ */
+exports.getTasks = async function getTasks(parameters) {
+  const { context, attempts, workflowSid, assignmentStatus, ordering } =
+    parameters;
+
+  if (!isNumber(attempts))
+    throw "Invalid parameters object passed. Parameters must contain the number of attempts";
+  if (!isObject(context))
+    throw "Invalid parameters object passed. Parameters must contain context object";
+
+  try {
+    const client = context.getTwilioClient();
+    const tasks = await client.taskrouter
+      .workspaces(process.env.TWILIO_FLEX_WORKSPACE_SID)
+      .tasks.list({ limit: 1000, workflowSid, assignmentStatus, ordering });
```

Comment:
Good find! All this time I have been tweaking the `limit` and `pageSize` parameters from the calling function - only to now notice that they are unused. Guess I need to re-perform that load testing 😬

Reply:
Done. Eliminated `pageSize`since it appears to not be supported by the Task resource anyway, and just went with `limit`. Now that this is no longer unwittingly hardcoded to 1000, I now see some more expected load test results with respect to task lookups failing when limit is < CPS of the load test. So bumping it back up to 50 to cater to a greater range of potential CPS settings.